sqlair.go

```diff
@@ -250,18 +265,22 @@ func (iter *Iterator) Close() error {
 }
 
 // Outcome holds metadata about executed queries, and can be provided as the
-// first output argument to any of the Get methods.
+// first output argument to any of the Get methods to populate it with
+// information about the query execution.
 type Outcome struct {
 	result sql.Result
 }
 
+// Result returns a sql.Result containing information about the query
+// execution. If no result is set then Result returns nil.
 func (o *Outcome) Result() sql.Result {
 	return o.result
 }
 
 // GetAll iterates over the query and scans all rows into the provided slices.
 // sliceArgs must contain pointers to slices of each of the output types.
-// An &Outcome{} variable may be provided as the first output variable.
+// An &Outcome{} struct may be provided as the first output variable to get
```

Comment:
I think we should be consistent with "Outcome struct" / "&Outcome{}" / "pointer to ..."